lib/tasks/scheduler.rake

```diff
@@ -1,10 +1,3 @@
-desc "Transition youth who are now 14, run by heroku scheduler"
-task transition_youth: :environment do
```

Comment:
@FireLemons this will make qa and prod scheduler fail, we should remove that when this is deployed